nftables_test.go

```diff
@@ -1620,8 +1620,191 @@ func TestConfigureRange(t *testing.T) {
 			&expr.Range{
 				Op:       expr.CmpOpNeq,
 				Register: 1,
-				FromData: (2024 << 16),
-				ToData:   (2030 << 16),
+				FromData: binaryutil.BigEndian.PutUint16(uint16(2024)),
+				ToData:   binaryutil.BigEndian.PutUint16(uint16(2030)),
+			},
+			// [ immediate reg 0 return ]
+			&expr.Verdict{
+				Kind: expr.VerdictKind(unix.NFT_RETURN),
+			},
+		},
+	})
+
+	if err := c.Flush(); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestConfigureRangeIPv4s(t *testing.T) {
```

Comment:
Here and below, why “IPv4s” instead of “IPv4”? What does the extra “s” stand for?

Reply:
it was just indicating that there are multiple IPs, removed 's'